### ISSUE TITLE
Accessibility errors found in your template

### ISSUE THREAD

**Issue by wapyce** [lddtime/jekyll-theme-php-net#1]
The user @carlsonsantana validate your site template "https://lddtime.github.io/jekyll-theme-php-net/" and found these accessibility errors:
* Page https://lddtime.github.io/jekyll-theme-php-net/:
  * Issue *8fd3aa7c-8c59-44de-bc3d-b1faedfc7821*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputSearch.Name
    * **Message**: This searchinput element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.
    * **Selector**: `#topsearch > input:nth-child(2)`
    * **Context**: `<input type="search" name="pattern" class="search-query" placeholder="搜索" accesskey="s">`
  * Issue *2fcf1be1-8230-4b2a-a047-f0758f232a1f*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle3.Guideline3_2.3_2_2.H32.2
    * **Message**: This form does not contain a submit button, which creates issues for those who cannot submit the form using the keyboard. Submit buttons are INPUT elements with type attribute "submit" or "image", or BUTTON elements with type "submit" or omitted/invalid.
    * **Selector**: `#topsearch`
    * **Context**: `<form class="navbar-search" id="topsearch" action="/search.php">
      <input type="hidden" nam...</form>`
  * Issue *b305f46a-18e5-4a20-bd64-47a908071f2a*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.4:1. Recommendation: change background to #000107.
    * **Selector**: `#head-nav > div:nth-child(1) > ul > li > a`
    * **Context**: `<a href="/jekyll-theme-php-net/about/">About</a>`
  * Issue *4d330632-bab7-4684-85b6-4117f5f54c4a*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
    * **Message**: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.
    * **Selector**: `#mainmenu-toggle`
    * **Context**: `<input type="checkbox" id="mainmenu-toggle">`
  * Issue *c887aefb-b7ad-424e-bc6f-3699d0cd1b7f*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputCheckbox.Name
    * **Message**: This checkboxinput element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.
    * **Selector**: `#mainmenu-toggle`
    * **Context**: `<input type="checkbox" id="mainmenu-toggle">`

You can check these accessibility errors using [pa11y](https://github.com/pa11y/pa11y).
You can view the [full validation results in our website](https://wapyce.herokuapp.com/validation/validations/9ad31114-98f0-4926-a096-5c0e248aceae/).
